Codeinwp/otter-blocks
# Remove Template Library

Closes #990.


### Summary

Removes Template Library entirely.

### Test instructions

- When adding a new section, the option to add a template shouldn't show anymore. 
- We need to make sure it doesn't break anything else.



---- 

### Checklist before the final review
